save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/vulnerability/VulnerabilityController.kt

```diff
@@ -101,6 +101,25 @@ class VulnerabilityController(
         ResponseEntity.ok("Vulnerability was successfully updated")
     }
 
+    @DeleteMapping("/delete")
+    @Operation(
+        method = "Delete",
+        summary = "Deleted vulnerability.",
+        description = "Deleted vulnerability.",
+    )
+    @ApiResponse(responseCode = "200", description = "Successfully deleted vulnerability")
+    @PreAuthorize("hasRole('ROLE_SUPER_ADMIN')")
+    fun delete(
+        @RequestParam name: String,
+        authentication: Authentication,
+    ): Mono<StringResponse> = blockingToMono {
+        if (authentication.hasRole(Role.SUPER_ADMIN)) {
+            vulnerabilityService.delete(name)
+        }
```

Comment:
Why do you need to double check the role?

Reply:
Yes, didn't notice it. Thank you